pkg/cluster/internal/providers/common/logs.go

```diff
@@ -41,6 +41,10 @@ func CollectLogs(n nodes.Node, dir string) error {
 			n.Command("journalctl", "--no-pager", "-u", "containerd.service"),
 			"containerd.log",
 		),
+		execToPathFn(
+			n.Command("crictl", "images"),
+			"containerd-images.log",
```

Comment:
might be better to pin the containerd endpoint with -r here and not rely on crictl's iteration through known endpoints.

Reply:
@BenTheElder is the expert on the storage thing ^ ^^

Reply:
nit: since this output is from crictl and shouldn't be containerd specific if we ever had another runtime, can we call it something more generic like images.log or crictl-images.log ?

Reply:
Done!